Flaky ControlledActorClockEndpointIT.testOffsetTime

**Summary**

- How often does the test fail? once
- Does it block your work? fails the build
- Do we suspect that it is a real failure? no, there was a connection problem

**Failures**

<details><summary>Example assertion failure</summary>
<pre>
[2022-09-07T00:20:54.880Z] [ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 30.331 s <<< FAILURE! - in io.camunda.zeebe.it.management.ControlledActorClockEndpointIT
[2022-09-07T00:20:54.880Z] [ERROR] io.camunda.zeebe.it.management.ControlledActorClockEndpointIT.testOffsetTime  Time elapsed: 17.673 s  <<< ERROR!
[2022-09-07T00:20:54.880Z] java.io.IOException: There was a problem while connecting to 127.0.0.1:36267
[2022-09-07T00:20:54.880Z] 	at org.testcontainers.shaded.com.trilead.ssh2.Connection.connect(Connection.java:755)
[2022-09-07T00:20:54.880Z] 	at org.testcontainers.containers.PortForwardingContainer.createSSHSession(PortForwardingContainer.java:51)
[2022-09-07T00:20:54.880Z] 	at org.testcontainers.containers.PortForwardingContainer.getSshConnection(PortForwardingContainer.java:27)
[2022-09-07T00:20:54.880Z] 	at org.testcontainers.containers.PortForwardingContainer.exposeHostPort(PortForwardingContainer.java:72)
[2022-09-07T00:20:54.880Z] 	at org.testcontainers.containers.PortForwardingContainer.exposeHostPort(PortForwardingContainer.java:66)
[2022-09-07T00:20:54.880Z] 	at org.testcontainers.Testcontainers.exposeHostPorts(Testcontainers.java:14)
[2022-09-07T00:20:54.880Z] 	at io.zeebe.containers.ZeebeBrokerNode.withDebugExporter(ZeebeBrokerNode.java:99)
[2022-09-07T00:20:54.880Z] 	at io.zeebe.containers.engine.DebugReceiverStream.lambda$start$0(DebugReceiverStream.java:76)
[2022-09-07T00:20:54.880Z] 	at java.base/java.util.Collections$SingletonSet.forEach(Collections.java:4905)
[2022-09-07T00:20:54.880Z] 	at io.zeebe.containers.engine.DebugReceiverStream.start(DebugReceiverStream.java:76)
[2022-09-07T00:20:54.881Z] 	at io.zeebe.containers.engine.ZeebeContainerEngine.start(ZeebeContainerEngine.java:97)
[2022-09-07T00:20:54.881Z] 	at org.testcontainers.junit.jupiter.TestcontainersExtension$StoreAdapter.start(TestcontainersExtension.java:242)
[2022-09-07T00:20:54.881Z] 	at org.testcontainers.junit.jupiter.TestcontainersExtension$StoreAdapter.access$200(TestcontainersExtension.java:229)
[2022-09-07T00:20:54.881Z] 	at org.testcontainers.junit.jupiter.TestcontainersExtension.lambda$null$4(TestcontainersExtension.java:81)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.execution.ExtensionValuesStore.lambda$getOrComputeIfAbsent$4(ExtensionValuesStore.java:86)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.execution.ExtensionValuesStore$MemoizingSupplier.computeValue(ExtensionValuesStore.java:223)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.execution.ExtensionValuesStore$MemoizingSupplier.get(ExtensionValuesStore.java:211)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.execution.ExtensionValuesStore$StoredValue.evaluate(ExtensionValuesStore.java:191)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.execution.ExtensionValuesStore$StoredValue.access$100(ExtensionValuesStore.java:171)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.execution.ExtensionValuesStore.getOrComputeIfAbsent(ExtensionValuesStore.java:89)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.execution.NamespaceAwareStore.getOrComputeIfAbsent(NamespaceAwareStore.java:53)
[2022-09-07T00:20:54.881Z] 	at org.testcontainers.junit.jupiter.TestcontainersExtension.lambda$beforeEach$5(TestcontainersExtension.java:81)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ReferencePipeline$15$1.accept(ReferencePipeline.java:540)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1845)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:960)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:934)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.AbstractTask.compute(AbstractTask.java:327)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:754)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:686)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateParallel(ReduceOps.java:927)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
[2022-09-07T00:20:54.881Z] 	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
[2022-09-07T00:20:54.881Z] 	at org.testcontainers.junit.jupiter.TestcontainersExtension.beforeEach(TestcontainersExtension.java:84)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeEachCallbacks$2(TestMethodTestDescriptor.java:166)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeMethodsOrCallbacksUntilExceptionOccurs$6(TestMethodTestDescriptor.java:202)
[2022-09-07T00:20:54.881Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeMethodsOrCallbacksUntilExceptionOccurs(TestMethodTestDescriptor.java:202)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeEachCallbacks(TestMethodTestDescriptor.java:165)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:132)
[2022-09-07T00:20:54.881Z] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ForkJoinPoolHierarchicalTestExecutorService$ExclusiveTask.compute(ForkJoinPoolHierarchicalTestExecutorService.java:185)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ForkJoinPoolHierarchicalTestExecutorService.executeNonConcurrentTasks(ForkJoinPoolHierarchicalTestExecutorService.java:155)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ForkJoinPoolHierarchicalTestExecutorService.invokeAll(ForkJoinPoolHierarchicalTestExecutorService.java:135)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ForkJoinPoolHierarchicalTestExecutorService$ExclusiveTask.compute(ForkJoinPoolHierarchicalTestExecutorService.java:185)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ForkJoinPoolHierarchicalTestExecutorService.invokeAll(ForkJoinPoolHierarchicalTestExecutorService.java:129)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
[2022-09-07T00:20:54.882Z] 	at org.junit.platform.engine.support.hierarchical.ForkJoinPoolHierarchicalTestExecutorService$ExclusiveTask.compute(ForkJoinPoolHierarchicalTestExecutorService.java:185)
[2022-09-07T00:20:54.882Z] 	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
[2022-09-07T00:20:54.882Z] 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
[2022-09-07T00:20:54.882Z] 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
[2022-09-07T00:20:54.882Z] 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
[2022-09-07T00:20:54.882Z] 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
[2022-09-07T00:20:54.882Z] 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
[2022-09-07T00:20:54.882Z] Caused by: java.io.IOException: Premature connection close
[2022-09-07T00:20:54.882Z] 	at org.testcontainers.shaded.com.trilead.ssh2.transport.ClientServerHello.readLineRN(ClientServerHello.java:33)
[2022-09-07T00:20:54.882Z] 	at org.testcontainers.shaded.com.trilead.ssh2.transport.ClientServerHello.<init>(ClientServerHello.java:68)
[2022-09-07T00:20:54.882Z] 	at org.testcontainers.shaded.com.trilead.ssh2.transport.TransportManager.initialize(TransportManager.java:457)
[2022-09-07T00:20:54.882Z] 	at org.testcontainers.shaded.com.trilead.ssh2.Connection.connect(Connection.java:699)
[2022-09-07T00:20:54.882Z] 	... 89 more
[2022-09-07T00:20:54.883Z] 
</pre>
</details>

**Hypotheses**

There was a connection problem during the test:
>There was a problem while connecting to 127.0.0.1:36267

We should verify that it isn't a problem when this happens, and potentially make the test more resilient to deal with this case.

**Logs**

<details><summary>Logs</summary>
<pre>
[2022-09-07T00:28:15.723Z] [INFO] Results:
[2022-09-07T00:28:15.723Z] [INFO] 
[2022-09-07T00:28:15.723Z] [WARNING] Flakes: 
[2022-09-07T00:28:15.723Z] [WARNING] io.camunda.zeebe.it.management.ControlledActorClockEndpointIT.testOffsetTime
[2022-09-07T00:28:15.723Z] [ERROR]   Run 1: ControlledActorClockEndpointIT.testOffsetTime » IO There was a problem while connecting to 127.0.0.1:36267
[2022-09-07T00:28:15.723Z] [INFO]   Run 2: PASS
[2022-09-07T00:28:15.723Z] [INFO] 
[2022-09-07T00:28:15.723Z] [INFO] 
[2022-09-07T00:28:15.723Z] [WARNING] Tests run: 237, Failures: 0, Errors: 0, Skipped: 0, Flakes: 1
[2022-09-07T00:28:15.724Z] [INFO] 
</pre>
</details>

--- a/parent/pom.xml
+++ b/parent/pom.xml
@@ -84,7 +84,7 @@
     <version.asm>9.3</version.asm>
     <version.testcontainers>1.17.3</version.testcontainers>
     <version.netflix.concurrency>0.3.6</version.netflix.concurrency>
-    <version.zeebe-test-container>3.5.1</version.zeebe-test-container>
+    <version.zeebe-test-container>3.5.2</version.zeebe-test-container>
     <version.feel-scala>1.15.2</version.feel-scala>
     <version.dmn-scala>1.7.3</version.dmn-scala>
     <version.rest-assured>5.2.0</version.rest-assured>

